assets/persistence/example_data.gd

```diff
@@ -0,0 +1,22 @@
+extends PersistentDataSection
+
+const PD_SECTION_EXAMPLE = "example"
+const PD_SECTION_EXAMPLE_TIME = "time"
```

Comment:
Could we make these field name definitions an enum, or array that we can do the below logic as a map instead of copy pasting for each field?